rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -177,6 +211,7 @@ def convert_to_data_format(
     ],
     zero_features: Optional[Dict[Text, List["Features"]]] = None,
     consider_dialogue_dimension: bool = True,
+    featurizers: Optional[List[Text]] = None,

```

Comment:
diet also uses `convert_to_data_format` but don't use this option. Does it make sense to unify the way ted and diet use featurizers?

Reply:
would do that as part of the bigger refactoring